docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -165,47 +165,50 @@ Test launcher script `7zip/tests/test.sh`:
 
 ```bash
 #!/bin/sh
+set -euo pipefail
```

Comment:
This is a take-it-or-leave-it suggestion as the ordering here doesn't actually matter,    but it would feel more readable to me to have this after the usage comment block.

Reply:
Fixed.  Thanks @smacfarlane!